src/pu-package.c

```diff
@@ -58,17 +100,25 @@ pu_package_create(gchar **files,
             return FALSE;
         }
 
-        if (g_str_equal(file, "layout.yaml"))
-            has_layout_yaml = TRUE;
+        if (strstr(file, ".yaml"))
+            layout_yaml_count++;
     }
 
-    if (!has_layout_yaml) {
+    if (!layout_yaml_count) {
         g_set_error(error, PU_PACKAGE_ERROR, PU_PACKAGE_ERROR_MISSING_LAYOUT,
                     "Package input files do not contain a layout configuration "
-                    "named 'layout.yaml'");
+                    "named '*.yaml'");
```

Comment:
Instead of using `'*.yaml'` (with the wildcard), describe it a little differently:
```
Package input files do not contain a layout configuration file with .yaml extension
```
or even simplify it:
```
Package input files do not contain a layout configuration file
```